prbt_support/urdf/prbt_macro.xacro

```diff
@@ -33,6 +33,10 @@ limitations under the License.
 
     <!-- robot foot link -->
     <link name="${prefix}base_link">
+        <inertial>
+          <mass value="2.4"/>
+          <inertia ixx="1e-3"  ixy="0"  ixz="0" iyy="1e-3" iyz="0" izz="1e-3" />
```

Comment:
Why not set inertia ixx, iyy, izz to zero?

Reply:
Even so `base_link` does not move (for most cases) it is not wrong to have an inertia. Probably the only case where this would matter is if an user adds another moving joint before the arm.